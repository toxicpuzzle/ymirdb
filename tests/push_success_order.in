SET a 1 2 3
SET b 2 3
PUSH a 3 2 1 b 
PUSH b 3 2 1
LIST ENTRIES
bye