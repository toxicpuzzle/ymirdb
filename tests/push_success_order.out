> ok

> ok

> ok

> ok

> b [1 2 3 2 3]
a [b 1 2 3 1 2 3]

> bye
